app/src/generated/graphql.schema.json

```diff
@@ -1623,6 +1623,16 @@
             },
             "defaultValue": null
           },
+          {
```

Comment:
i'm not sure about some of the changes in this file... did something with `game_state` change recently?